README.md

```diff
@@ -77,136 +72,97 @@ usefull additions to your .gitignore
+- [Breakout](https://pmp-p.github.io/pygame-breakout-wasm/index.html)
+- [PyChess](https://pmp-p.github.io/pygame-pychess-wasm/index.html)
+- [Penguins Can't Fly !](https://pmp-p.github.io/pygame-PenguinsCantFly-wasm/)
+- [John's Adventure](https://pmp-p.github.io/pygame-JohnsAdventure-wasm/)
+- [3D Tic-Tac-Toe](https://pmp-p.github.io/pygame-ttt-3d-wasm/)
+- [Arachnoids](https://pmp-p.github.io/pygame-arachnoids-wasm/)
+- [Sudoku Solver](https://www.pete-j-matthews.com/Sudoku-Solver/)
 
-Please ! use the tag [pygame-wasm](https://github.com/topics/pygame-wasm) for your projects hosted on Github
-and also add a favicon.png icon 32x32 next to your game main.py, so it is picked up by default.
+Source code for these games can be found [here](https://github.com/pmp-p?tab=repositories&q=pygame-.-wasm&sort=name). You can tag your Github repositories with [[pygame-wasm]](https://github.com/topics/pygame-wasm).
 
 ### Script demos
 
-nb : code is read-only, prefer right-click then open in new window.
-
-[i18n bidi,complex scripts](/showroom/pypad_git.html?-i#src/test_hb.py)
-
-[camera](/showroom/pypad_git.html?-i#src/test_vidcap.py)
-
-[Panda3D](/showroom/pypad_dev.html?-i#src/test_panda3d_cube.py)
-
-[Audio Record/Play](/showroom/pypad_dev.html?-i#src/test_audio.py)
-
-[Html output](/showroom/pypad_dev.html?-i#src/test_html.py)
-
-
-___
-
-## Technology:
-
-[initial devlog](https://github.com/pygame/pygame/issues/718) 
-[pygame-ce devlog](https://github.com/pygame-community/pygame-ce/issues/540)
-
-[Python Wasm explained by core dev Christian Heimes (youtube video)](https://www.youtube.com/watch?v=oa2LllRZUlU)
+The code is read-only, so you should right-click then open in a new window.
 
+- [i18n bidi, complex scripts](/showroom/pypad_git.html?-i#src/test_hb.py)
+- [Camera](/showroom/pypad_git.html?-i#src/test_vidcap.py)
+- [Panda3D](/showroom/pypad_dev.html?-i#src/test_panda3d_cube.py)
+- [Audio Record/Play](/showroom/pypad_dev.html?-i#src/test_audio.py)
+- [HTML output](/showroom/pypad_dev.html?-i#src/test_html.py)
 
-### Early demos from above talk, may not fully work as intended :)
+## Technology
 
-[pygame tech demo PyCon DE & PyData Berlin 2022](https://pmp-p.github.io/pygame-wasm/)
+- [Initial discussion](https://github.com/pygame/pygame/issues/718) 
+- [Discussion at pygame-ce repo](https://github.com/pygame-community/pygame-ce/issues/540)
+- [Python-WASM explained by core dev Christian Heimes (video)](https://www.youtube.com/watch?v=oa2LllRZUlU)
 
-[Galaxy Attack](https://pmp-p.github.io/pygame-galaxy-attack-wasm/)
+### Early demos from above talk, may not work as intended :)
 
+- [Pygame tech demo PyCon DE & PyData Berlin 2022](https://pmp-p.github.io/pygame-wasm/)
+- [Galaxy Attack](https://pmp-p.github.io/pygame-galaxy-attack-wasm/)
 
-French : Python WebAssembly at PyCon FR 2023
+Python WebAssembly at PyCon FR 2023 (in French): 
 [Pour quoi, pour qui et comment](https://harfang3d.github.io/pyconfr2023/#1)
 
-## current status
```

Comment:
Please keep these blocks separated and do not change urls
status / support / connect